src/node_sqlite.cc

```diff
@@ -211,6 +219,53 @@ void DatabaseSync::Exec(const FunctionCallbackInfo<Value>& args) {
   CHECK_ERROR_OR_THROW(env->isolate(), db->connection_, r, SQLITE_OK, void());
 }
 
+void DatabaseSync::EnableLoadExtension(
+    const FunctionCallbackInfo<Value>& args) {
+  DatabaseSync* db;
+  ASSIGN_OR_RETURN_UNWRAP(&db, args.This());
+  Environment* env = Environment::GetCurrent(args);
+  if (!args[0]->IsBoolean()) {
+    node::THROW_ERR_INVALID_ARG_TYPE(
```

Comment:
There is a PR open to remove the `node` namespace usage in this file, so we might as well do that here. There are a few other instances in this PR.

```suggestion
    THROW_ERR_INVALID_ARG_TYPE(
```